tests/ast/test_ast_setup.py

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+from snowflake.snowpark.mock._connection import MockServerConnection
+
+
+def test_session(session):
```

Comment:
we prob. can remove this, I think I understand why you want to test for this. It's a check that we have a valid configuration. If you feel strongly about keeping it, I'd suggest to rename
```
test_session -> test_ensure_valid_test_setup()
```
or so. It doesn't really test the "session" itself, but simply checks its internals.